Party List Click To Target

Rave7283 opened the pull request:
- Added config option to enable click-to-target on party members in party list
  - Defaults to off to maintain consistency with previous version
- Added click-to-target invisible button to party list entries for party members
  - Button is only created if the config option is enabled and the party member is in zone
- Renamed a couple variables in ../partylist/display.lua to convey that they represent the entry length/height

https://github.com/user-attachments/assets/91693c95-57e2-40cb-afe1-5c215d2aaca8

Rave7283 committed:
Party List Click To Target

Shuu-37 commented:
I'm a bit worried about renaming allBarsLength - because we use that naming  in a lot of our other files..  @Rave7283

Shuu-37 committed:
Fix indentation and revert variable rename

- Revert entryWidth back to allBarsLengths (keep original name)
- Keep entrySize → entryHeight rename
- Fix mixed tabs/spaces indentation to use 4 spaces
- Fix typo: "an party member" → "a party member"

Shuu-37 committed:
Merge branch 'main' of github.com:tirem/HXUI into Party-List-Click-To-Target

Shuu-37 committed:
Merge branch 'main' of github.com:tirem/HXUI into Party-List-Click-To-Target
